tensorboard/webapp/widgets/data_table/data_table_header_component.scss

```diff
@@ -26,4 +27,14 @@ limitations under the License.
 mat-icon {
   height: 12px;
   width: 12px;
+  min-width: 12px;
+  line-height: 12px;
+}
+
+.step-at-min {
+  min-width: 37px; // This was imperically tested to be the minimum with that forces "step at min" to wrap to only 2 lines.
```

Comment:
Since you are quoting the string "step at min" and we're referring to the width of the text, where capitalization actually matters, might as well capitalize it consistent with what appears to the user:

"Step at Min"

(same for "Step at Max")

Reply:
Good call.